protos/gripper/gripper.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package mavsdk.rpc.gripper;
+
+option java_package = "io.mavsdk.gripper";
+option java_outer_classname = "GripperProto";
+
+service GripperService {
+    /*
+     * Gripper grab cargo.
+     */
+    rpc Grab(GrabRequest) returns(GrabResponse) {}
+
+    /*
+     * Gripper release cargo.
+     */
+    rpc Release(ReleaseRequest) returns(ReleaseResponse) {}
+}
+
+enum GripperAction {
+    GRIPPER_ACTION_RELEASE = 0;
+    GRIPPER_ACTION_GRAB = 1;
+}
+
+message GrabRequest {
+    uint32 instance = 1;
+}
+message GrabResponse {
+    GripperResult gripper_result = 1;
+}
+
+message ReleaseRequest {
+    uint32 instance = 1;
+}
+message ReleaseResponse {
+    GripperResult gripper_result = 1;
+}
+
+// Result type.
+message GripperResult {
+    // Possible results returned for action requests.
+    enum Result {
+        RESULT_UNKNOWN = 0; // Unknown result
+        RESULT_SUCCESS = 1; // Request was successful
+        RESULT_NO_SYSTEM = 2; // No system is connected
+        RESULT_CONNECTION_ERROR = 3; // Connection error
+        RESULT_BUSY = 4; // Vehicle is busy
+        RESULT_COMMAND_DENIED = 5; // Command refused by vehicle
+        RESULT_COMMAND_DENIED_LANDED_STATE_UNKNOWN = 6; // Command refused because landed state is unknown
+        RESULT_COMMAND_DENIED_NOT_LANDED = 7; // Command refused because vehicle not landed
+        RESULT_TIMEOUT = 8; // Request timed out
+        RESULT_VTOL_TRANSITION_SUPPORT_UNKNOWN = 9; // Hybrid/VTOL transition support is unknown
+        RESULT_NO_VTOL_TRANSITION_SUPPORT = 10; // Vehicle does not support hybrid/VTOL transitions
+        RESULT_PARAMETER_ERROR = 11; // Error getting or setting parameter
+        RESULT_UNSUPPORTED = 12; // Action not supported
```

Comment:
Same here: does that happen for the gripper?

Reply:
I think it could if the gripper commands are not implemented, e.g. for an older PX4 version.

Reply:
`RESULT_UNSUPPORTED` is one of the few that makes sense

Reply:
Would it make sense to support _all_ of the possible results that the command sender can return?

https://github.com/mavlink/MAVSDK/blob/444d6b693b2d76cd594b1df074f28f468cc78f47/src/mavsdk/core/mavlink_command_sender.h#L24-L37

In case of simple systems like winches and grippers, the results `NoSystem`, `ConnectionError`, and `Timeout` for example are basically the same.

Reply:
Aha, it seems that the grpc code auto-generation at least expects there to be a `NoSystem` result, so I can't remove that one.

Reply:
They are different though.